app/mutations/FormChange/updateProjectFormChange.ts

```diff
@@ -0,0 +1,27 @@
+import { graphql } from "react-relay";
+
+/**
+ *
+ * Specific mutation for a form_change for a project.
+ * Includes whether the suggested proposal reference is unique in the system.
+ *
+ */
+
+const mutation = graphql`
+  mutation updateProjectFormChangeMutation($input: UpdateFormChangeInput!) {
+    updateFormChange(input: $input) {
+      formChange {
+        id
+        newFormData
+        operation
+        isUniqueValue(columnName: "proposalReference")
```

Comment:
I don't think we actually need any of the other fragments, since this mutation is specific for a project form change, there won't be any updates to the contact form changes or the manager form changes